customer-wallet/src/styles/variables.scss

```diff
@@ -1,15 +0,0 @@
-$mainColor: #2F3240;
```

Comment:
We should keep these around anway

Reply:
Yea, plan to use the MuiTheme though for it